web/packages/teleport/src/Navigation/Section.tsx

```diff
@@ -519,6 +526,30 @@ function LicenseFooter({
   );
 }
 
+export function PoweredByTeleportLogo() {
+  const theme = useTheme();
+  const src = theme.type === 'dark' ? logoPoweredByDark : logoPoweredByLight;
+  return (
+    <StyledFooterBox
+      py={3}
+      px={4}
+      pb={4}
+      css={`
+        border: none;
+      `}
+    >
+      <Image
+        src={src}
+        width="fit-content"
+        alt="powered by teleport logo"
```

Comment:
"logo" is redundant for screen readers.
```suggestion
        alt="Powered by Teleport"
```